ote/config.edn

```diff
@@ -8,4 +8,7 @@
  :places {:finnish-municipalities
           {:type :shapefile
            :name-field "NAMEFIN"
-           :url "https://s3.eu-central-1.amazonaws.com/ote-assets/places/finnish-municipalities/SuomenKuntajako_2017_10k.shp"}}}
+           :url "https://s3.eu-central-1.amazonaws.com/ote-assets/places/finnish-municipalities/SuomenKuntajako_2017_10k.shp"}}
+
+ :nap {:api "http://localhost:8080/api/"
```

Comment:
Do we need new config file for staging environment? Or how we can change these adressess to production and staging?

Reply:
Yes, config.edn for deployed env is at aws/ansible/templates/ote/config.edn.j2, I'll update that as well.